map.go

```diff
@@ -195,9 +219,7 @@ func (r *rover) GenerateModuleMap(parent *Resource, parentModule string) {
 
```

Comment:
if you change lines 212 to the following it resolves the error I mention in https://github.com/im2nguyen/rover/pull/79#issuecomment-1010603958

```
else if rs.Type == ResourceTypeResource {
	r := configs[parentConfig].Module.ManagedResources[ind]

	if r != nil {
		fname = filepath.Base(r.Pos.Filename)
		re.Line = &r.Pos.Line
	}
}
```